<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recipes_Viewer</class>
 <widget class="QWidget" name="Recipes_Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>632</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1270</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="topMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Recipes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="btn_add">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QListView" name="list_recipes">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Recipe Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCustomPlot" name="viz_cost_price_history" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCustomPlot" name="viz_volatility_history" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QChartView" name="viz_recipe_composition">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>461</width>
            <height>288</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="6" column="3" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_ingredient_category">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Ingredient Name:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Volatility:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_recipe_name">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_recipe_price">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_recipe_category">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Category:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Category</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_recipe_cost">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Recipe Name:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Recipe Price:</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_recipe_volatility">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Cost:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_ingredient_name">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Supplier:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_ingredient_supplier">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Brand:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_ingredient_brand">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Ingredient Price:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_ingredient_price">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Yield (UOM)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_ingredient_yield_uom">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Volatility:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_ingredient_volatility">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Item Number:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_ingredient_number">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="3" alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_ingredient_description">
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
